src/api/app/views/webui/shared/_error.html.haml

```diff
@@ -0,0 +1 @@
+= flash[:error]
```

Comment:
This file was introduced here and removed in the third commit "Remove useless partial". That commit should be sqashed with the first commit.